src/uct/api/uct.h

```diff
@@ -1205,26 +1198,28 @@ struct uct_ep_params {
     unsigned                            path_index;
 
     /**
-     * Callback is called when the client side endpoint is bound to network
-     * device and provides related info as defined in
-     * @ref uct_cm_ep_resolve_args_t. This field is only valid if
-     * @ref uct_ep_params_t::sockaddr is set.
-     * @note It is never guaranteed that the callaback will be called. If, for
-     * example, the endpoint goes into error state before issuing the connection
-     * request, the callback will not be invoked.
+     * This callback is invoked when the remote server address provided in field
+     * @ref uct_ep_params_t::sockaddr is resolved to the local device to be used
+     * for connection establishment.
+     * @note In case of a connection error, this callback will not be invoked;
+     *       @ref uct_ep_params_t::sockaddr_cb_client with indicating error
+     *       status will be invoked instead.
+     * @note This field is mutually exclusive with
+     *       @ref uct_ep_params.sockaddr_pack_cb.
      */
-    uct_cm_ep_resolve_callback_t        sockaddr_resolve_cb;
+    uct_cm_ep_resolve_callback_t        cm_resolve_cb;
 
     /**
-     * User's private data to be passed from server to client, can be
-     * initialized only together with @a conn_request, can not be initialized
-     * together with @a sockaddr_pack_cb.
+     * Private data to be passed from server to client. Can be used only along
+     * with @ref uct_ep_params.conn_request.
+     * @note This field is mutually exclusive with
+     *       @ref uct_ep_params.sockaddr_pack_cb.
      */
-    void                                *private_data;
+    const void                          *private_data;
 
     /**
-     * Length of @a private_data, the maximal allowed value is indicated by the
-     * @ref uct_cm_attr::max_conn_priv.
+     * Length of @ref uct_ep_params.private_data, the maximal allowed value is
+     * indicated by the @ref uct_cm_attr::max_conn_priv.
```

Comment:
`@ref uct_cm_attr.max_conn_priv`

Reply:
returned back all `@ref` to fields to `::` for consistency